tests/unittests/test_distro_detection.py

```diff
@@ -0,0 +1,33 @@
+import os
+import sys
+
+from distgen.distro_version import detect_default_distro
+
+from mock import patch
+
+class TestDistroDetection(object):
+
+    @patch('subprocess.Popen')
+    @patch('platform.dist')
+    def test_rpm(self, dist, popen):
+        for arch in [b'x86_64', b'i386']:
```

Comment:
For testing multiple similar scenarios, I'd suggest using `pytest.mark.parametrize`. It makes tests cleaner and executes them as separate test cases. See e.g. https://github.com/devexp-db/distgen/blob/master/tests/unittests/test_commands.py#L18

Reply:
yay, thanks for the point!